lib/manager/npm/post-update/index.ts

```diff
@@ -360,13 +435,20 @@ export async function getAdditionalFiles(
   }
   for (const lockFile of dirs.npmLockDirs) {
     const lockFileDir = path.dirname(lockFile);
+    const fullLockFileDir = upath.join(config.localDir, lockFileDir);
+    const npmrcContent = await getNpmrcContent(fullLockFileDir);
+    await updateNpmrcContent(
```

Comment:
Does this logic allow the for the scenario where there's *no* `.npmrc` content or additional content? e.g. where `npmrcContent` is null?

Reply:
Yes this should be handled. The [test I did on Github](https://github.com/renovatebot/renovate/pull/5164#issuecomment-575977801) would have been on repository that didn't have an npmrc and that was successful.